models/document-generation.ts

```diff
@@ -420,7 +421,7 @@ const descriptionOfWork = {
 export const generateDocumentSchema = {
   type: "object",
   properties: {
-    documentType: { enum: [DocumentType.DESCRIPTION_OF_WORK] },
+    documentType: { enum: [DocumentType.DESCRIPTION_OF_WORK, DocumentType.INDEPENDENT_GOVERNMENT_COST_ESTIMATE] },
     templatePayload: descriptionOfWork,
```

Comment:
#1105 contains many related improvements to this file.  This is just the bare minimum to get the 501.